examples/Example/Page/Todos/Todo.hs

```diff
@@ -26,23 +27,17 @@ page = do
         space
         el (Style.h1 . pad 10) "Todos"
         space
-      hyper AllTodos $ todosView FilterAll todos
+      hyper MkTodosView $ todosView FilterAll todos
 
-simplePage :: (Todos :> es) => Eff es (Page '[AllTodos, TodoView])
-simplePage = do
-  todos <- Todos.loadAll
-  pure $ do
-    hyper AllTodos $ todosView FilterAll todos
-
---- AllTodos ----------------------------------------------------------------------------
+--- TodosView ----------------------------------------------------------------------------
 
-data AllTodos = AllTodos
+data TodosView = MkTodosView
```

Comment:
Careful not to rename / refactor things to your preference in this PR. It'll make it hard to separate merging the example from a discussion on preferred naming.

Reply:
Yes, I said in the description

> Currently, I've renamed some types, e.g. AllTodos => MkTodosView to facilitate my own understanding, but I'll revert this at the end.

It looks like you didn't see it

Reply:
Got it thanks!